pkg/backend/httpstate/journal/snapshot.go

```diff
@@ -80,11 +85,13 @@ func (j *cloudJournaler) AddJournalEntry(entry engine.JournalEntry) error {
 		return fmt.Errorf("serializing journal entry: %w", err)
 	}
 
-	result := make(chan error, 1)
 	j.entries <- saveJournalEntry{
 		entry:  serialized,
 		result: result,
 	}
+	if entry.ElideWrite {
```

Comment:
Feel like this should contract.Assert that result is nil

Reply:
added :heavy_check_mark: